.github/workflows/go.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v5
       with:
-        go-version: 1.24.0
+        go-version: 1.23.0
```

Comment:
any reason for this change?

Reply:
Right I don't need that anymore :) reverted